src/processTargets/modifiers/SmallPaintStage.ts

```diff
@@ -0,0 +1,66 @@
+import { zip } from "lodash";
+import { Target } from "../../typings/target.types";
+import {
+  ContainingScopeModifier,
+  EveryScopeModifier,
+} from "../../typings/targetDescriptor.types";
+import { ProcessedTargetsContext } from "../../typings/Types";
+import { ModifierStage } from "../PipelineStages.types";
+import { TokenTarget } from "../targets";
+import { NonWhitespaceSequenceStage } from "./scopeTypeStages/RegexStage";
+import { InteriorOnlyStage } from "./InteriorStage";
+
+export type NonWhitespaceOrQuoteSequenceModifier = (
+  | ContainingScopeModifier
+  | EveryScopeModifier
+) & {
+  scopeType: { type: "nonWhitespaceOrQuoteSequence" };
+};
+
+/**
+ * Intersection of NonWhitespaceSequenceStage and InteriorOnlyStage
+ * Expand the target until reaching a white space or surrounding pair.
+ * If there is no surrounding pair defaults to the non white space sequence
+ */
+export default class SmallPaintStage implements ModifierStage {
+  constructor(private modifier: NonWhitespaceOrQuoteSequenceModifier) {}
+
+  run(context: ProcessedTargetsContext, target: Target): Target[] {
+    const paintStage = new NonWhitespaceSequenceStage({
```

Comment:
I have mixed feelings about instantiating the stage directly.  Wouldn't it be better not to tightly couple ourselves to the class that implements the stage?  We have `getModifierStage`, so I'm not sure why we don't want to use it? cc/ @AndreasArvidsson

Reply:
Just less abstraction. If that is good or not is something has probably varies from person to person.

Reply:
I've updated this first instance to use getModifierStage, and the second one to call processSurroundingPair directly.